src/messages/responses/cache-list-fetch.ts

```diff
@@ -0,0 +1,33 @@
+import {SdkError} from '../../errors/errors';
+import {ResponseBase} from './response-base';
+import {applyMixins, ErrorBody} from '../../errors/error-utils';
+
+export abstract class Response extends ResponseBase {}
+
+export class Hit extends Response {
+  private readonly _values: Uint8Array[];
+  constructor(values: Uint8Array[]) {
+    super();
+    this._values = values;
+  }
+
+  public values(): Uint8Array[] {
```

Comment:
Should we rename this to `valueListUint8Array`? 
That's how it's named in the .NET. 
https://github.com/momentohq/client-sdk-dotnet-incubating/blob/main/src/Momento.Sdk.Incubating/Responses/CacheListFetchResponse.cs#L34

Also that way, we don't need to use `_` [here](https://github.com/momentohq/client-sdk-javascript/pull/149/files#diff-d6754ba5098f5e6fb3b8604bc5ceaeca7fdbe136912365868476fe9cccdaf768R8).